src/events/OmniboxEvents.ts

```diff
@@ -29,6 +30,14 @@ export interface IOmniboxPreprocessResultForQueryEventArgs {
   result: Result;
 }
 
+export interface IBuildingQuerySuggestionsArgs {
```

Comment:
What about `IBuildingQuerySuggestArgs` and `IQuerySuggestSuccessArgs`? Little bit shorter, I also thing "suggest(ions)" should be singular, with "args" remaining plural. It would be more consistent with other names.

e.g. line 34 `IQuerySuggestRequest`, line 39 `IQuerySuggestCompletion`, `IQuerySuggestAddon`.